thunder/tests/test_core.py

```diff
@@ -2969,12 +2969,12 @@ def fn(x):
     jfn = thunder.jit(fn)
     actual_dtype = jfn(x)
 
-    assert actual_dtype == thunder.dtypes.float32
+    assert actual_dtype == torch.float32
```

Comment:
In this and the tests below (the above is special and excepmpt), could you adopt the pattern of
`assert fn(x) == jfn(x)` please?